cmd/install/install.go

```diff
@@ -161,11 +162,13 @@ func NewCommand() *cobra.Command {
 	opts.MetricsSet = metrics.DefaultMetricsSet
 	opts.EnableConversionWebhook = true // default to enabling the conversion webhook
 	opts.ExternalDNSImage = ExternalDNSImage
+	opts.HCDestroyGracePeriod = 0
 
 	cmd.PersistentFlags().StringVar(&opts.Namespace, "namespace", "hypershift", "The namespace in which to install HyperShift")
 	cmd.PersistentFlags().StringVar(&opts.HyperShiftImage, "hypershift-image", version.HyperShiftImage, "The HyperShift image to deploy")
 	cmd.PersistentFlags().BoolVar(&opts.Development, "development", false, "Enable tweaks to facilitate local development")
 	cmd.PersistentFlags().BoolVar(&opts.EnableDefaultingWebhook, "enable-defaulting-webhook", false, "Enable webhook for defaulting hypershift API types")
+	cmd.PersistentFlags().DurationVar(&opts.HCDestroyGracePeriod, "destroy-grace-period", opts.HCDestroyGracePeriod, "Grace period for HostedCluster to be deleted. The format is duration; e.g. 30s or 1h30m45s; 0 means no grace period")
```

Comment:
Why is this an operator level setting?

Reply:
Basically to avoid modify the Spec in a Bug. IMHO the best way to implement this, is as a new API to allow you to change it for every cluster, but the feature freeze date is here and as a first implementation I think is ok.

If I can do that during this period I will follow that way.

Reply:
Talking with @csrwng we agree on use an annotation and remove the flag implementation, this way is better for SD. WDYT @enxebre ?